src/Gui/DlgSettingsDocument.ui

```diff
@@ -433,6 +433,9 @@ automatically run a file recovery when it is started.</string>
       </item>
       <item row="1" column="0">
        <widget class="Gui::PrefCheckBox" name="prefPartialLoading">
+        <property name="toolTip">
+         <string>Needs a tooltip!!!</string>
```

Comment:
Needs a tooltip

Reply:
That's from the link branch.. @realthunder ?

Reply:
Suggested tooltip: Enable partial loading of external linked document. When enabled, only the referenced objects and their dependencies will be loaded when a linked document is auto opened together with the main document. A partially loaded document cannot be edited. Double click the document icon in the tree view to reload it in full.

A more detailed explanation of the feature can be found [here](https://github.com/realthunder/FreeCAD_assembly3/wiki/Core-Changes#partial-document-loading)

Reply:
Thanks @realthunder  
I don't think we can enable hyperlinks in tooltips, AFAICT..?  
I've added your feedback + explicitly show the URL. I wish there was a better way to do this.  
748755b

Reply:
No, I don't mean to add the URL in the tooltip. I just put it here for you guys, in case you want to know more. Or maybe put the link somewhere in the wiki.

Reply:
realthunder is correct, luzpaz. The tooltip is meant to be short; expanded information should be given in the actual documentation pages, in this case, the wiki; see [Preferences Editor](https://www.freecadweb.org/wiki/Preferences_Editor).

Reply:
How do we go about tracking the need to document these needs? The workflow is very poor in this regards. Can we think think about it and figure out a way?
  
Also documentation-wise (this has come up before (TODO: find link)) we have differences between version 0.18 and 0.19 so 'preferences editor' GUI updates aren't shown graphically, their only represented textually at this time. Is that the best we can do for now until 0.19 is released or should we have a separate section of 'Preferences Editor' that accommodates 0.19 pages (with 0.19 screenshots?)

Reply:
Forum thread: [Requesting a workflow the indicates if documentation is obsolete or needs updating or is changing in the next release?](https://forum.freecadweb.org/viewtopic.php?f=21&t=39810).

Obviously having diverging documentation in 0.18 and 0.19 is not ideal, and could be done better. We just don't have the manpower to handle it. But like everything in free software, you can do it yourself luzpaz, if you have the time and inclination.

As I stated in that thread, I think at this point in time we don't have enough resources to look into every detail; some things are bound to be obsolete, or not applicable depending on the version. I would focus on 0.19 documentation. People who use 0.18 may ignore the new options.

A good way to gauge whether the documentation is correct or not is by looking at the forum. If forum users claim that they can't find the options, and that everything is confusing, then that means there is a clear incentive to improve what is written; but so far I haven't read this, so it seems to me it isn't a very big problem. This should be the strongest indication that the current information is good enough.

Reply:
Thanks @vocx-fc. Removed the URL and will update the wiki text for now without the screenshot GUI.